### ISSUE TITLE
Groups broken after clicking "Leave Group"

### ISSUE THREAD

**Issue by playfulpachyderm** [tloncorp/landscape-apps#2611]
Groups is broken after clicking "Leave Group" from a group I had open.  I can't open the groups app at all anymore, it just shows this error.

```
TypeError: Cannot read properties of undefined (reading 'title')
    at Cl ([my-ship-domain]/apps/groups/assets/index.b18819a0.js:48:176000)
    at Kl ([my-ship-domain]/apps/groups/assets/react-beautiful-dnd.fde4d763.js:19:52884)
    at Ka ([my-ship-domain]/apps/groups/assets/react-beautiful-dnd.fde4d763.js:19:62361)
    at mf ([my-ship-domain]/apps/groups/assets/react-beautiful-dnd.fde4d763.js:23:9604)
    at pf ([my-ship-domain]/apps/groups/assets/react-beautiful-dnd.fde4d763.js:23:947)
    at yg ([my-ship-domain]/apps/groups/assets/react-beautiful-dnd.fde4d763.js:23:875)
    at rn ([my-ship-domain]/apps/groups/assets/react-beautiful-dnd.fde4d763.js:23:730)
    at sl ([my-ship-domain]/apps/groups/assets/react-beautiful-dnd.fde4d763.js:21:10884)
    at [my-ship-domain]/apps/groups/assets/react-beautiful-dnd.fde4d763.js:19:38716
    at e.unstable_runWithPriority ([my-ship-domain]/apps/groups/assets/react-beautiful-dnd.fde4d763.js:8:3811)
```

**To Reproduce**
I am using Port.  I get the error every time I open Groups now.  Refreshing the page and restarting Port didn't help, it's still there.  It was originally triggered by clicking "Leave Group" though.

**Expected behavior**
Groups should open without an error.

**Additional context:**
- Ship type: planet
- Instance type: self hosted on cloud, using Port as a client to it
- OS/browser: Port

**Comment by playfulpachyderm** [comment]
Also it would be good to update the behavior of this button to pre-fill an issue with the bug template instead of just the stack trace
![image](https://github.com/tloncorp/landscape-apps/assets/5721742/0561f14c-2f48-410a-808e-3bc9fdcf0f6e)

**Comment by jamesacklin** [comment]
Fixed by #2595
